src/array.cr

```diff
@@ -190,7 +190,7 @@ class Array(T)
     size <=> other.size
   end
 
-  # Set intersection: returns a new `Array` containing elements common to `self`
+  # Sets intersection: returns a new `Array` containing elements common to `self`
```

Comment:
This is actually a noun, not a verb. ("Set intersection", as in the intersection of two sets)

Same below on the next diff line for this file